cub/cub/device/device_copy.cuh

```diff
@@ -188,7 +188,7 @@ struct DeviceCopy
       SizeIteratorT,
       RangeOffsetT,
       BlockOffsetT,
-      detail::DeviceBatchMemcpyPolicy<RangeOffsetT, BlockOffsetT>,
+      detail::batch_memcpy::policy_hub<RangeOffsetT, BlockOffsetT>,
```

Comment:
Question: I am not seing the include added here, is that intentional?

Reply:
The explicit include wasn't there before. It's brought in transitively via `#include <cub/device/dispatch/dispatch_batch_memcpy.cuh>`

Reply:
Added the include.